### ISSUE TITLE
Printer-level output options

### ISSUE THREAD

**Issue by kageurufu** [supermerill/SuperSlicer#2862]
**Is your feature request related to a problem? Please describe.**

I would like to be able to just set global output options instead of having to make sure I copy changes to every print profile

**Describe the solution you'd like**

I want to add a copy of the `Output options` pane from `Print Settings` to the `Printer Settings` tab. 

Specifically, `Output file`, `Other`, and `Post-processing script` are the most desired, although I could see value in having some of the `Sequential printing` options at a printer level as well.

**Describe how it would work**

The Printer level settings would be defaults, unless overridden on Print settings. Some things like the Extruder clearance could also be moved outright. 

`Post-processing scripts` could be a little more nuanced, with `Print settings` getting `Pre-processing scripts` and (needs a better name) `After post-processing scripts`. These would run in order, Printer -> Pre-processing, Print -> Post-processing, then Printer -> After post-processing. My use case here would be to put `preprocess_cancellation` in the Printer Pre-processing, any print-specific modification scripts like Ellis' old acceleration control on the print settings level, then finally `klipper_estimator` in the printer-level `After post-processing`. This also makes it easier to share print settings profiles across a variety of physical printers, where klipper_estimator and other similar tools might need printer-specific settings (moonraker urls, specific acceleration controls, etc).

**Describe alternatives you've considered**

I've considered writing a python script that would ensure output options are consistent across all print settings through ini editing, but its a hack at best.

Custom variables for some of these can work too, but again would be fragile based on needing to keep the output options synchronized across profiles.

**Comment by LazeMSS** [comment]
Agreed - like the https://github.com/Annex-Engineering/klipper_estimator would be nice to have on a printer level
